src/rendering/contexts/native/lambda.zig

```diff
@@ -134,15 +134,16 @@ test "isLambdaInvoker" {
 /// fn (*TData, LambdaContext) anyerror!void
 pub fn isValidLambdaFunction(comptime TData: type, comptime TFn: type) bool {
     const fn_info = switch (@typeInfo(TFn)) {
-        .Fn => |info| info,
+        .@"fn" => |info| info,
         else => return false,
     };
 
-    const Type = std.builtin.Type;
-    const FnParam = Type.Fn.Param;
+    // TODO: Update types to oncemore be strict on the input type
+    //const Type = std.builtin.Type;
+    //const FnParam = Type.@"fn".Param;
```

Comment:
Can you motivate why this TODO?

Reply:
Apologies, I should've explained that better. If I change the lines such that lines 141-146 read
```zig
    // TODO: Update types to oncemore be strict on the input type
    const Type = std.builtin.Type;
    const FnParam = Type.@"fn".Param;
    
    const paramIs = struct {
        fn action(comptime param: FnParam, comptime types: []const type) bool {
```
Then upon running `zig build test` I am met with a plethora of errors all stating
```
src/rendering/contexts/native/lambda.zig:143:31: error: type '@typeInfo(builtin.Type).@"union".tag_type.?' does not support field access
    const FnParam = Type.@"fn".Param;
                    ~~~~~~~~~~^~~~~~
zig/lib/std/builtin.zig:257:18: note: enum declared here
pub const Type = union(enum) {
                 ^~~~~
src/rendering/contexts/native/invoker.zig:199:90: note: called from here
                            const is_valid_lambda = comptime lambda.isValidLambdaFunction(TValue, @TypeOf(bound_fn));
                                                             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~
```

I was not sure how exactly to handle this and thus marked it as something to work on in the future. I'll be sure to be more clear about that next time and apologize for any confusion I may have caused.

Reply:
Ah, in this case we don't need to change `Fn` to `@fn`, as we do need to access the type, not the union field.
See https://ziglang.org/documentation/master/std/#std.builtin.Type.Fn.Param

Can you send another PR fixing it?